src/adapters/adkernel.js

```diff
@@ -179,16 +211,32 @@ const AdKernelAdapter = function AdKernelAdapter() {
   }
 
   /**
+   * Evaluate creative size from response or from request
+   */
+  function getCreativeSize(imp, bid) {
+    let dimensions = (bid.h && bid.w) ? bid : (imp.banner || imp.video);
+    return {
+      w: dimensions.w,
+      h: dimensions.h
+    };
+  }
+
+  /**
    *  Create bid object for the bid manager
    */
   function createBidObject(resp, bid, width, height) {
-    return Object.assign(bidfactory.createBid(1, bid), {
+    let bidObj = Object.assign(bidfactory.createBid(1, bid), {
       bidderCode: bid.bidder,
-      ad: formatAdMarkup(resp),
       width: width,
       height: height,
       cpm: parseFloat(resp.price)
     });
+    if (bid.params.video) {
+      bidObj.vastUrl = resp.nurl;
```

Comment:
Please also signify this is a video bid response with `bidObj.mediaType = 'video';`, and you may want to add `bidObj.descriptionUrl = resp.nurl;`, which is used in `pbjs.buildMasterVideoTagFromAdserverTag`